config/webpack/webpack.config.js

```diff
@@ -38,11 +38,17 @@ const webpackConfig = (options, env, argv) => {
         {
           test: /\.(scss|css)$/,
           use: [
-            MiniCssExtractPlugin.loader,
+            {
+              loader: MiniCssExtractPlugin.loader,
+              options: {
+                hmr: !production, // only enable hot module reloading in development
```

Comment:
This isn't the default?

Reply:
I guess I didn't look into the code of the loader, but the docs called out this:
https://github.com/webpack-contrib/mini-css-extract-plugin#hot-module-reloading-hmr

Reply:
By default it looks like this is always off:
- https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/src/loader.js#L60

Then,
https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/src/loader.js#L189